README.md

```diff
@@ -51,6 +51,8 @@ By default the config is read from `config.json` in the project root, this can b
   - A map of LD operations to map to DevCycle operations
   - DevCycle operations: `=`, `!=`, `>`, `<`, `>=`, `<=`, `contain`, `!contain`, `exist`, `!exist`
   - Equivalent env var: OPERATION_MAP
+- <b>provider</b>: <i>string</i>
+  - The provider to import the feature flags from
```

Comment:
we should document the current options, so that this doesn't confuse users of the importer